### DISCUSSION TITLE
How does unregister work with field array inputs?

### DISCUSSION THREAD

**Discussion by wdfinch** [react-hook-form/react-hook-form#4510] [Q&A] [answered]
It seems like the `unregister` method does nothing when we pass it a field array input. For example, if I do:

`unregister(`array${index}.field`)`, then nothing happens. Could someone explain if something is happening and how this works?

**Comment by bluebill1049** [comment, marked as answer]
did you unmount that input as well? otherwise it will get registered again.

**Reply by wdfinch** [reply]
No, I was trying to use it to reset one input in a field array. It seems like I need to use `setValue` with the initial input for that.
